src/components/AuthoritiesLookup/AuthoritiesLookup.js

```diff
@@ -59,38 +58,33 @@ const AuthoritiesLookup = ({
   const [isFilterPaneVisible, setIsFilterPaneVisible] = useState(true);
   const [showDetailView, setShowDetailView] = useState(false);
 
-  const { navigationSegmentValue, filters } = useContext(AuthoritiesSearchContext);
+  const { filters } = useContext(AuthoritiesSearchContext);
   const [, setSelectedAuthorityRecordContext] = useContext(SelectedAuthorityRecordContext);
 
   const toggleFilterPane = () => setIsFilterPaneVisible(!isFilterPaneVisible);
 
   const closeDetailView = () => {
     setShowDetailView(false);
-    setSelectedAuthorityRecordContext(null);
+  };
+
+  const openDetailView = () => {
+    setSelectedAuthorityRecordContext(authorities[0]);
```

Comment:
Isn't the `onOpenRecord` prop in `SearchResultList` called with a record argument? We should call `setSelectedAuthorityRecordContext` with that object instead

Reply:
Thanks, fixed